composer.json

```
@@ -21,28 +21,28 @@
     "prefer-stable": true,
     "require": {
         "php": "^8.0",
-        "cycle/annotated": "^3.0",
-        "cycle/migrations": "^3.0",
-        "cycle/orm": "^2.0",
+        "cycle/annotated": "^3.2.1",
+        "cycle/migrations": "^3.1.1",
+        "cycle/orm": "^2.2.2",
         "cycle/schema-migrations-generator": "^2.0",
-        "cycle/schema-renderer": "^1.0",
+        "cycle/schema-renderer": "^1.2",
         "psr/event-dispatcher": "^1.0",
         "psr/simple-cache": "^2.0|^3.0",
         "spiral/attributes": "^2.7|^3.0",
         "webmozart/glob": "^4.0",
         "yiisoft/aliases": "^1.1|^2.0|^3.0",
         "yiisoft/data": "^1.0",
-        "yiisoft/friendly-exception": "^1.0",
+        "yiisoft/friendly-exception": "^1.1",
         "yiisoft/injector": "^1.1",
         "yiisoft/yii-console": "^2.0"
     },
     "require-dev": {
         "cycle/entity-behavior": "^1.0",
         "jetbrains/phpstorm-attributes": "^1.0",
-        "phpunit/phpunit": "^9.5",
+        "phpunit/phpunit": "^9.6 || ^10",
```

```suggestion
        "phpunit/phpunit": "^9.6|^10",
```

https://github.com/yiisoft/docs/blob/master/000-packages.md#composerjson